src/test/java/hudson/plugins/warnings/parser/SdccParserTest.java

```diff
@@ -0,0 +1,114 @@
+package hudson.plugins.warnings.parser;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Iterator;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import hudson.plugins.analysis.util.model.FileAnnotation;
+import hudson.plugins.analysis.util.model.Priority;
+
+/**
+ * Tests the class {@link SdccParser}.
+ *
+ * @author Kay van der Zander
+ */
+public class SdccParserTest extends ParserTester {
+    private static final String TYPE = new SdccParser().getGroup();
+
+    /**
+     * Parses a file with warnings/errors in all styles. it check the amount of error/warnings found
+     *
```

Comment:
Please also check for checkstyle warnings

Reply:
That is done by the other tests.  First test is for the amount it should detect with the output.txt file.
Then the same file is used to check on every fault that is in there. if the style is correct.

Reply:
Sorry, I mean *you* should run CheckStyle, PMD and FindBugs on your source code. Then you will see, that there are a lot of warnings, e.g. a sentence in a comment has to be finished by using a point ;-)

Reply:
ah i understand. oke i will.